src/cart/swim/swim.c

```diff
@@ -372,7 +373,7 @@ swim_member_dead(struct swim_context *ctx, swim_id_t from, swim_id_t id, uint64_
 		}
 	}
 
-	SWIM_ERROR("member %lu is DEAD\n", id);
+	SWIM_ERROR("member %lu %lu is DEAD\n", id, nr);
```

Comment:
any reason why dead members are error printed while alive/suspect is info?

Reply:
Sorry, forgot to publish the reply above yesterday.